communication/message_formatting.md

```diff
@@ -4,12 +4,25 @@ Message Formatting
 - **3.5.1** ![](/badge/req.png) A client must encode emoticons as Unicode emoji.
 
 - **3.5.2** ![](/badge/req.png) A receiving client will make a best-effort to display
-  text exactly as it was sent, and must not transform the text in any way.
+  text exactly as it was intended, and must not materially change the text in any way.
     - The above rule does not apply if malformed UTF-8 text is received. In
       this case, a client may transform the text to any extent necessary to
       make it displayable.
+    - A material change is any change that likely will change the structure, layout,
+      content, meaning, text, or intent of the message.
 
-- **3.5.3** ![](/badge/req.png) A sending client must not insert or remove any bytes from
-  the original input.
+- **5.3.3** ![](/badge/rec.png) It's a strong suggestion that messaging clients support 
+  at the very least a subset of the common markdown syntax. The markdown used in this 
+  standard is a subset of Github Flavored Markdown. With the exclusion of direct 
```

Comment:
[**@GrayHatter**](https://github.com/GrayHatter) commented on [3. Jan. 2016, 20:57 MEZ](https://github.com/Tox/Tox-Client-Standard/pull/25#discussion_r48696389 "2016-01-03T19:57:58Z - Replied by Github Comment Enhancer"):
> That way if I only want to implement `**bold** /italics/ _underline_` I can just stop there.

The issue is `_underline_` e.g. conflicts with your spec, because if `_underline_` should be underlined you cannot do this, because in GFM it looks like this: _underline_